CMakeLists.txt

```diff
@@ -1,29 +1,38 @@
-cmake_minimum_required(VERSION 2.8.12)
+#TODO
+# - fix / modernize cmake
+# - lower cmake version a bit ~3.7 to support older systems
+# - fix abseil warnings or try to use SYSTEM / -isystem includes
+
+cmake_minimum_required(VERSION 3.8)
 project(s2-geometry)
+
+#produces compile_command.json
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+
+#not used?
+#list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/third_party/cmake")
+#include(CheckCXXCompilerFlag)
+#include(FindPackageHandleStandardArgs)
+
+## set options
 include(CMakeDependentOption)
-include(CheckCXXCompilerFlag)
 include(FeatureSummary)
-include(FindPackageHandleStandardArgs)
-
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/third_party/cmake")
 
 option(WITH_GFLAGS "Use gflags to change command line flags." OFF)
-add_feature_info(GFLAGS WITH_GFLAGS
-                 "allows changing command line flags.")
+add_feature_info(GFLAGS WITH_GFLAGS "allows changing command line flags.")
```

Comment:
Don't reformat for no reason.

Reply:
Is it ok to reformat everything to your style when I am done?

Reply:
Sure. Is there a standard CMake indentation style?